tests/model/test_model.py

```diff
@@ -1205,13 +1205,60 @@ def test_update_star_status(self, mocker, model, event_op,
         changed_ids = set(indexed_ids) & set(event_message_ids)
         for changed_id in changed_ids:
             assert model.index['messages'][changed_id]['flags'] == flags_after
-        (model.update_rendered_view.
-         has_calls([mocker.call(changed_id) for changed_id in changed_ids]))
+            assert model.update_rendered_view.has_calls(
+                    [mocker.call(changed_id)])
```

Comment:
I've pushed this line inside the loop, since it's basically the same. Also added the assert statement.

Reply:
Not really, a similar fix that has been applied in `test_update_unread_status` needs to be applied here. Would it be more appropriate to fix this in a different commit/PR ?